R/evalPars.R

```diff
@@ -3,6 +3,64 @@
 ## Copyright (c) Joseph Guillaume <josephguillaume@gmail.com>
 ##
 
+
+
+#' Evaluate a model for a matrix of parameters
+#'
+#' \code{evalPars} evaluates a model for a named matrix of parameters.
+#'
+#' \code{evalPars} is used in conjunction with \code{getFreeParsRanges}
+#' and the \code{sensitivity} package to perform sensitivity analysis. See
+#' \code{demo(sensitivity)}. Note that the objective function may more
+#' generally return any scalar result, e.g. a scalar prediction calculated only
+#' using X.
+#'
+#' Individual model evaluations are generally very fast, so parallelisation is
+#' only really worthwhile when large numbers of evaluations are needed.
+#' \code{"clusterApply"} has a slightly lower overhead. \code{"foreach"} allows
+#' a broader range of options.
+#'
+#' @importFrom iterators iter
```

Comment:
```suggestion
```
No longer needed now that we're using iterators::iter?

Reply:
Have removed this and can confirm that it works locally. Should be in the push I just uploaded (apologies, forgot you'd made a suggestion, so committed it on my end an uploaded)